test/RefBackend/mlprogram-bufferize.mlir

```diff
@@ -0,0 +1,30 @@
+// RUN: torch-mlir-opt %s -refback-mlprogram-bufferize -split-input-file | FileCheck %s
+
+// CHECK-LABEL:   memref.global "private" @global_seed : memref<i64> = dense<0>
+// CHECK-LABEL:   func.func @forward() -> i64 {
+// CHECK:           %[[CST127:.*]] = arith.constant 127 : i64
```

Comment:
test the case of an ml_program.global with 1) a non-tensor type and 2) a tensor type that cannot be converted to memref.